daemon/snap.go

```diff
@@ -200,5 +200,10 @@ func mapRemote(remoteSnap *snap.Info) map[string]interface{} {
 	if len(remoteSnap.Prices) > 0 {
 		result["prices"] = remoteSnap.Prices
 	}
+
+	if len(remoteSnap.ChannelMap) > 0 {
+		result["channel-map"] = remoteSnap.ChannelMap
```

Comment:
related to the other point, the thing is called the channel map, which is why I called it that here. Calling the channel map "channels" would be confusing, both here and in speaking about it.

I understand what you say about calling it a map though, and I have no problem with you renaming the thing (by talking with people and coming to an agreement with them that the thing should be called whatever it is), at which point it makes sense to call it something else here.